bin/autodoc-cli

```diff
@@ -15,66 +15,30 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-# Create appendices/command-ref.rst for inclusion in HTML doc.
+# Create command help transcripts for inclusion in HTML doc.
 
 # All paths relative to 'doc/src/custom/' directory:
-COMMAND_REF_FILE="$(dirname "$0")/../src/appendices/command-ref.rst"
-CYLC="cylc"
+COMMAND_REF_FILE="$(dirname "$0")/../src/reference/command-help.rst"
 
 cat > "$COMMAND_REF_FILE" <<END
 .. _CommandReference:
 
 Command Reference
 =================
 
-.. _help:
-
-Help
-----
-
-.. code-block:: none
-
-   $("${CYLC}" --help | awk '{print "   " $0}')
-
-Command Categories
-------------------
-
-END
-
-for CAT in $($CYLC categories); do
-	cat >> "$COMMAND_REF_FILE" <<END
-
-.. _command-cat-${CAT}:
-
-${CAT}
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code-block:: none
-
-   $("$CYLC" "$CAT" --help | awk '{print "   " $0}')
-
-END
-
-done
-
-cat >> "$COMMAND_REF_FILE" <<END
-
-Commands
---------
-
 END
 
-for COM in $($CYLC commands); do
+for COM in $(cylc help all | awk '{print $1}' | sort); do
```

Comment:
CI build failed, I think because this line doesn't work if the terminal width is too low ?? (newlines get inserted into the help text)

Reply:
Would have to be narrower than the longest command name to trip awk up?!